space/cart2polarXYZ.glsl

```diff
@@ -0,0 +1,16 @@
+/*
+description: Returns the cartesian to polar coordinates
+use: spherical(<vec3> st)
+*/
+
+#ifndef FNC_CART2POLARXYZ
+#define FNC_CART2POLARXYZ
+vec3 cart2polarXYZ( in vec3 st ) 
```

Comment:
sorry maybe I didn't express correctly. **Overloading** means that have the same exact name but the arguments are different. In this case this fuction will have the same `cart2polar` name in the `cart2polar.glsl` but will be different because they have different argument types. Like for example: https://github.com/patriciogonzalezvivo/lygia/blob/main/math/bump.glsl#L14

Reply:
Sorry, I don't have a lot of experience with overloading functions.  So instead of adding new functions I should edit the existing files to add new version, correct?